Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+fhBuildNode {
```

Comment:
@grdryn Should we be setting any particular version of node to use here?

Reply:
@mikenairn I originally thought about using node6, since that's what our components use. That doesn't currently work though, as I think the version of grunt-fh-build needs to be upgraded, and maybe other stuff. Also, since many of these modules get included in cloud apps, they need to be usable with Node.js 4, and if we're not building/testing with all of our node versions, then maybe oldest supported is a good start.